pandas/io/clipboard/__init__.py

```diff
@@ -322,19 +335,6 @@ def __setattr__(self, key, value):
 def init_windows_clipboard():
     global HGLOBAL, LPVOID, DWORD, LPCSTR, INT
     global HWND, HINSTANCE, HMENU, BOOL, UINT, HANDLE
-    from ctypes.wintypes import (
```

Comment:
let's add `ctypes` to the `BLOCKLIST` and revert this change

Reply:
Added ctypes to BLOCKLIST in scripts/standard_library_imports_should_be_global.py

Reply:
can you revert the change too please?